recover_handler.go

```diff
@@ -15,11 +15,6 @@ func Recover(handler http.Handler) http.Handler {
 			defer func() {
 				if rec := recover(); rec != nil {
 					msg, stack := extractPanic(rec)
-					log.WithFields(log.Fields{
```

Comment:
Worse, the service would shut down without this :D It would be logged, but not using our logger.